### PR TITLE
Initially completed the bitcoin wallet connection

### PR DESCRIPTION
**Pull request by wow-sven**
## Summary

If you want to verify it. `ppm i && pnpm rooch-sdk gen && pnpm rooch-sdk build && pnpm rooch-sdk-kit build && cd sdk/typescript/templates/react-viet && pnpm dev`

1. Add unisat, okx wallet support on bitcoin blockchain
2. All wrapped in rooch-sdk-kit package

TODO: 
1. address mapping
2. If the user's browser has both eth and okx wallet installed, there will be a conflict when using the eth network

- Closes #issue

### CODE REVIEW

File: crates/rooch-framework/sources/auth_validator/bitcoin_validator.move [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    }
+
+    fun rotate_authentication_key(ctx: &mut Context, account_addr: address, authentication_key: vector<u8>) {
+        account_authentication::rotate_authentication_key<BitcoinValidator>(ctx, account_addr, authentication_key);
+    }
+
+    public entry fun remove_authentication_key_entry(ctx: &mut Context, account: &signer) {
+        account_authentication::remove_authentication_key<BitcoinValidator>(ctx, signer::address_of(account));
+    }
+
```
Comment (jolestar) [inline crates/rooch-framework/sources/auth_validator/bitcoin_validator.move:52]: Bitcoin auth validator does not need to support the rotate authentication key.

File: crates/rooch-framework/sources/auth_validator/bitcoin_validator.move [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        option::extract(&mut get_authentication_key_option_from_account(ctx, addr))
+    }
+
+    /// Only validate the authenticator's signature.
+    public fun validate_signature(authenticator_payload: &vector<u8>, tx_hash: &vector<u8>) {
+        std::debug::print(tx_hash);
+        assert!(
+            ecdsa_k1_recoverable::verify(
+                &get_signature_from_authenticator_payload(authenticator_payload),
+                // &get_public_key_from_authenticator_payload(authenticator_payload),
```
Comment (jolestar) [inline crates/rooch-framework/sources/auth_validator/bitcoin_validator.move:117]: Do we need the public key?

File: crates/rooch-framework/sources/auth_validator/bitcoin_validator.move [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    }
+
+    fun pre_execute(
+        _ctx: &mut Context,
+    ) {}
+
+    fun post_execute(
+        ctx: &mut Context,
+    ) {
+        let account_addr = context::sender(ctx);
```
Comment (jolestar) [inline crates/rooch-framework/sources/auth_validator/bitcoin_validator.move:139]: We need to bind the address mapping here.

File: crates/rooch-framework/sources/auth_validator/bitcoin_validator.move (hunk 2 above)
Comment (wow-sven) [reply crates/rooch-framework/sources/auth_validator/bitcoin_validator.move]: do without

File: crates/rooch-framework/sources/auth_validator/bitcoin_validator.move (hunk 1 above)
Comment (wow-sven) [reply crates/rooch-framework/sources/auth_validator/bitcoin_validator.move]: I removed all the code associated with the authentication key

File: crates/rooch-framework/sources/auth_validator/bitcoin_validator.move (hunk 3 above)
Comment (wow-sven) [reply crates/rooch-framework/sources/auth_validator/bitcoin_validator.move]: yup

**Review by jolestar — APPROVED** [review]

**Pull request merged**